script.js

```diff
@@ -1,4 +1,172 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+// var main = function (input) {
+//   var myOutputValue = "hello world";
+//   return myOutputValue;
+// };
+
+/// 2 players taking turns, p1 and p2
+// upon submit, game rolls 2 dice , show 2 numbers, p1 choose 1 or 2 to combine number (eg 24 or 42)
+// store results for p1
+// p2 do the same ; store results for p2
+// compare p1 and p2
+//higher number wins
+
+// 4 game state - "P1 Dice Roll" , "P1 Choose Order" , "P2 Dice Roll" , "P2 Choose Order"
+
+/////////////////////
+
+var gameState = "P1 Roll Dice";
+
+var p1PlayerRollDice = [];
+var p2PlayerRollDice = [];
+var p1PlayerTotal = [];
+var p2PlayerTotal = [];
+
+function getRandomNum() {
+  return Math.floor(Math.random() * 6) + 1;
+}
+var randomNum = getRandomNum();
+console.log(randomNum);
+
+function p1RollDice() {
```

Comment:
Take a look at this function and the p2RollDice function. They look pretty much identical. The whole purpose of functions is to reduce code duplication. Try and see how you can achieve the same result with a single function. Also, try to use for loops whenever possible.